data/food/export_agb/ingredients_base.json

```diff
@@ -6,6 +6,7 @@
     "default_origin": "EuropeAndMaghreb",
     "raw_to_cooked_ratio": 0.974,
     "density": 1,
+    "transport_cooling": "none",
```

Comment:
No cooling at all for eggs? We sure of that? I'm especially surprised as eggs seem to be the only food from our list which is never cooled when transported, which sounds counter-intuitive.

Reply:
Eggs don't need cooling when stored, and I guess this means they don't either need it when transported? See the last table in [the docs](https://fabrique-numerique.gitbook.io/ecobalyse/alimentaire/transport#procedes-de-transport).